vllm/model_executor/models/mixtral_quant.py

```diff
@@ -0,0 +1,412 @@
+# coding=utf-8
```

Comment:
Instead of inserting a new file just for this, would it be better to add the MLP class again in `mixtral.py` and falling back to the original code for instances where linear method is None?

Reply:
Hi @AlpinDale, thank you for the comment. We are doing this to resolve release blocker. What you mentioned will be the right fix for the future.